coding-standards-and-naming.rst

```diff
@@ -229,6 +229,102 @@ happen with the agreement of the parties involved). This document does not try
 to provide a hard guideline on what constitutes a sufficiently important
 library. The application of common sense is recommended.
 
+Throwables
+==========
+
+Newly introduced extensions MUST follow the following rules, existing extensions
+SHOULD follow the rules for newly introduced exceptions, but MAY diverge for
+consistency with existing symbols.
+
+In the interest of readability the following text uses the word "exception" to
+refer to any kind of ``Throwable``. ``Exception`` in code blocks refers to the
+``\Exception`` class.
+
+The class name of an exception MUST end in ``Exception`` (when extending from
+the ``Exception`` hierarchy) or ``Error`` (for the ``Error`` hierarchy).
+Exceptions MUST NOT be ``final``.
```

Comment:
> Exceptions MUST NOT be ``final``.

I disagree with this, personally. If extensions have their own hierarchy of exceptions, why can't they choose to make them final as well?

Reply:
see: https://externals.io/message/127206#127256